.gitignore

```diff
@@ -276,6 +276,6 @@ scipy/optimize/_group_columns.c
 scipy/optimize/cython_optimize/_zeros.c
 scipy/optimize/cython_optimize/_zeros.pyx
 scipy/optimize/lbfgsb/_lbfgsbmodule.c
-scipy/optimize/_highs/cython/src/highs_wrapper.cxx
-scipy/optimize/_highs/cython/src/mpswriter.cxx
-scipy/optimize/_highs/cython/src/constants.cxx
+scipy/optimize/_highs/cython/src/_highs_wrapper.cxx
+scipy/optimize/_highs/cython/src/_mpswriter.cxx
+scipy/optimize/_highs/cython/src/_constants.cxx
```

Comment:
I think this is meant to be `_highs_constants.cxx` based on the renaming below, and the untracked file left behind when I build your feature branch locally:

```
On branch highs-constants-fix
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	scipy/optimize/_highs/cython/src/_highs_constants.cxx
```

Reply:
Good catch, fixed in latest push